Reverse ETL 2.0 feedback (#7015)
segmentio/segment-docs: src/connections/reverse-etl/reverse-etl-catalog.md

beta: false
---

> Reverse ETL supports the entire Segment destination catalog - Actions destinations are natively supported and all other destinations are supported through the [Segment Connections](#segment-connections-destination) destination. 

Author: ```suggestion
Reverse ETL supports the Actions destinations listed in this catalog. Most destinations not listed here are supported through the [Segment Connections](#segment-connections-destination) destination. 
```
